File: Programmers/Level1/약수의 개수와 덧셈/약수의 개수와 덧셈.py
def solution(left, right):
    answer = 0
    for i in range(left, right+1):
        cnt = 1
        tmp = 0
        for j in range(1, i//2 + 1):
            if i%j == 0:
                cnt += 1

        if cnt%2 != 0:
            tmp = i*(-1)
        else:
            tmp = i
            
        answer += tmp
        
    return answer

File: Programmers/Level1/약수의 개수와 덧셈/약수의 개수와 덧셈.java
class Solution {
    public int solution(int left, int right) {
        int answer = 0;
        for(int i=left; i<=right; i++){
            int cnt = 0;
            int tmp = 0;
            System.out.println(i);
            for(int j=1; j<=i/2; j++){
                if(i%j == 0){
                    cnt++;
                }
            }
 
            if((cnt+1)%2!=0){
                tmp = i*(-1);
            }else{
                tmp = i;
            }
            
            answer += tmp;
        }
        return answer;
    }
}
